packages/components/src/mobile/readable-content-view/index.native.js

```diff
@@ -3,25 +3,84 @@
  */
 import { View, Dimensions } from 'react-native';
 
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
-const ReadableContentView = ( { reversed, children, style } ) => (
-	<View style={ styles.container }>
-		<View
-			style={ [
-				reversed
-					? styles.reversedCenteredContent
-					: styles.centeredContent,
-				style,
-			] }
-		>
-			{ children }
+const PIXEL_RATIO = 2;
+
+const BREAKPOINTS = {
+	wide: 1024,
+	big: 820,
+	medium: 768,
+	small: 680,
+};
```

Comment:
I assume these BREAKPOINTS has to be different than [used for web](https://github.com/WordPress/gutenberg/blob/master/packages/viewport/src/index.js#L17), but I will recommend to:

1. Move it to utils (probably in other blocks we can have a need to use them as well 🤔)
2. Rename `big` to `large` to be consistent with web ✌️

Reply:
Yeah the breakpoints are very specific for mobile (not web) I'll move it to a utils file and update the name to `large` thanks!